src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -347,6 +348,7 @@
 
   &.pf-m-split-button {
     padding: 0; // pass padding to children
+    align-items: baseline;
```

Comment:
Also the menu toggle button/icon are going to be baseline aligned now instead of center. That's _probably_ alright since I don't know how often this would come up, and the typeahead menu toggle has its own alignment that aligns the toggle button to the center (and that's used for typeahead select). Tho if we can center align it, that would probably be good so this doesn't pop up as a bug later. It was center aligned in PF4, too.

PF4
<img width="824" alt="Screenshot 2023-07-14 at 1 27 11 PM" src="https://github.com/patternfly/patternfly/assets/35148959/5fca5948-54f0-4c8d-ac5b-eb7c50963b18">

This PR
<img width="831" alt="Screenshot 2023-07-14 at 1 27 17 PM" src="https://github.com/patternfly/patternfly/assets/35148959/84753458-de8a-4ba0-921a-5842e15995dc">